src/pipecat/transports/base_transport.py

```diff
@@ -33,7 +33,8 @@ class TransportParams(BaseModel):
     audio_out_channels: int = 1
     audio_out_bitrate: int = 96000
     audio_out_10ms_chunks: int = 4
-    audio_out_mixer: Optional[BaseAudioMixer] = None
+    audio_out_mixer: Optional[BaseAudioMixer | Mapping[Optional[str], BaseAudioMixer]] = None
+    audio_out_destinations: List[str] = []
```

Comment:
Should we have something equivalent for the `sources`, like `audio_in_sources`? 

Otherwise, I think the only way to handle this is to call `capture_participant_audio` multiple times directly on the transport to receive the custom audio tracks, right ?

Reply:
The same for video, maybe `video_in_sources` ?

Reply:
BTW, this is so cool, I need to add support for this custom tracks for the `SmallWebRTCTransport` as well. =)

Reply:
Yes, right now you have to call `capture_participant_audio`. That's what we currently need to do for video as well. I think this is fine, because you can select the framerate in the video case for example. If we add a `video_in_sources` it would be a bit more complicated.

Reply:
We also need to call `capture_participant_transcription` as well, so I guess it's common right now.

Reply:
And also, this is for participant. That is we want to selectively capture audio/video for each participant, not for all of them. Having `audio_in_sources` and `video_in_sources` wouldn't work to well.